projects/packages/my-jetpack/src/class-rest-product-data.php

```diff
@@ -68,7 +68,9 @@ public static function get_all_product_data() {
 
 		// If site has backups and the realtime backup capability, add latest undo event
 		if (
-			in_array( 'backup-realtime', $capabilities->data['capabilities'], true )
+			property_exists( $capabilities, 'data' )
+			&& isset( $capabilities->data['capabilities'] )
```

Comment:
`isset` alone does the right thing here.
```suggestion
			isset( $capabilities->data['capabilities'] )
```